hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -101,42 +104,57 @@ public Response get(
       @QueryParam("uploads") String uploads,
       @QueryParam("acl") String aclMarker,
       @Context HttpHeaders hh) throws OS3Exception, IOException {
+    S3GAction s3GAction = S3GAction.GET_BUCKET;
+    Map<String, String> auditParams = S3Utils.genAuditParam(
+        "bucket", bucketName,
+        "delimiter", delimiter,
+        "encoding-type", encodingType,
+        "marker", marker,
+        "max-keys", String.valueOf(maxKeys),
+        "prefix", prefix,
+        "browser", browser,
+        "continuation-token", continueToken,
+        "start-after", startAfter,
+        "uploads", uploads,
+        "acl", aclMarker
```

Comment:
I think we should omit `uploads` and `aclMarker`, they are not real parameters.  Only used to distinguish some operations, which are already indicated in the audit log by the `op` field.